app/notifications/admin/developers/new_celebration_package_request_notification.rb

```diff
@@ -0,0 +1,22 @@
+module Admin
+  module Developers
+    class NewCelebrationPackageRequestNotification < ApplicationNotification
+      deliver_by :database
+      deliver_by :email, mailer: "AdminMailer", method: :developers_celebration_package_request
+
+      param :form
```

Comment:
This should probably be renamed to match the new model.

Reply:
Renamed  in 9cf0ce13394fccf9b7d041e82417ba41d3d6c9b8